starlette_web/contrib/scheduler/apps.py

```diff
@@ -0,0 +1,39 @@
+import json
+
+import croniter
+from croniter.croniter import CroniterError
+
+from starlette_web.common.conf import settings as project_settings
+from starlette_web.common.conf.base_app_config import BaseAppConfig
+from starlette_web.common.http.exceptions import ImproperlyConfigured
+from starlette_web.contrib.scheduler.app_settings import Settings
+from starlette_web.contrib.scheduler.backends.base import JobType
+
+
+class AppConfig(BaseAppConfig):
+    app_name = "scheduler"
+
+    def perform_checks(self):
+        jobs_list = Settings(project_settings).PERIODIC_JOBS
+
+        for job in jobs_list:
+            try:
+                _ = json.JSONEncoder(sort_keys=True).encode(job)
+            except (TypeError, ValueError) as exc:
+                raise ImproperlyConfigured(
+                    message=f"Cannot json-encode job {job}",
+                    details=str(exc),
+                ) from exc
+
+            try:
+                assert len(job) == 5
+            except AssertionError:
+                raise ImproperlyConfigured(
+                    details=f"Required periodic job format in settings: {JobType}",
+                )
+
+            if job[0] != "@reboot":
+                try:
+                    _ = next(croniter.croniter(job[0]))
```

Comment:
```suggestion
                    next(croniter.croniter(job[0]))
```
\+ добавь тут плиз коммент почему именно мы должны вызывать next и что за магия творится) буквально в 2-3х словах

Reply:
Либа croniter генерирует генератор :) который бесконечно возвращает время следующего запуска таски по переданному шаблону. Здесь я просто дергаю генерацию 1 раз, чтобы посмотреть, упадет ли с ошибкой (из-за невалидного паттерна кронтаба) или нет

Например, от "* * * * 8" упадет, сказав, что значение для дня недели невалидное

Reply:
готово